Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -134,7 +171,47 @@ void APlayCppSdkActor::InitializeWalletConnect(
             });
 }
 
-void APlayCppSdkActor::RestoreClient(FString jsondata, bool &success,
+void APlayCppSdkActor::OnInitializeWalletConnect(bool succeed,
+                                                 FString message) {
+  FString succeed_result = succeed ? "true" : "false";
+  UE_LOG(LogTemp, Log, TEXT("Initialize Wallet Connect: %s %s"),
+         *succeed_result, *message);
+
+  // Setup Callback
+  bool IsSetupCallback;
+  FString SetupCallbackOutputMessage;
+  OnReceiveWalletconnectSessionInfoDelegate.BindDynamic(
+      this, &APlayCppSdkActor::OnWalletconnectSessionInfo);
+  SetupCallback(OnReceiveWalletconnectSessionInfoDelegate, IsSetupCallback,
+                SetupCallbackOutputMessage);
+  if (IsSetupCallback) {
+    FString GetConnectionStringOutput;
+    bool IsGetConnectionString;
+    FString GetConnectionStringOutputMessage;
+    GetConnectionString(GetConnectionStringOutput, IsGetConnectionString,
+                        GetConnectionStringOutputMessage);
+    if (IsGetConnectionString) {
+      UE_LOG(LogTemp, Log, TEXT("Connection String: "),
+             *GetConnectionStringOutput);
+      // Launch Crypto Wallet
+      UKismetSystemLibrary::LaunchURL(
+          GetCryptoWalletUrl(GetConnectionStringOutput));
```

Comment:
I guess on mobile, one will always launch the URL, on consoles, just display the QR code... on desktops, I guess one may have both / let the player choose?

Reply:
Done